### ISSUE TITLE
Rewrite to DRY code in the SDK installation test

### ISSUE THREAD

**Issue by rpannek** [Pelagicore/pelux-sde#6]
Right now the test_sdk_intstallation.py sets up every fixture from scratch, even though the only thing different is the name of the SDK file. Instead of copy&pasting it should be rewritten so adding the next fixture like that would be easy.

**Comment by gunnarx** [comment]
I didn't see *that much* duplication but a little.   If I understand which code you meant here in the same way as you, then I have this duplication removed and a patch queued up already.   Also I have a few commits that clean up whitespace, PEP8 issues and a few more things.

But everything is in a bit of flux right now - aiming to converge GDP SDE and this SDE, so I'll hold off on the PRs a little while.  Just so you know, a change proposal should be available for this issue and more.

**Comment by rpannek** [comment]
Ok good to hear!

**Comment by sashko** [comment]
@gunnarx could you please update a status on this issue? Tack.

**Comment by gunnarx** [comment]
I received an email about a mention on this task, but it's not here anymore so maybe you deleted.  Anyway, I don't really have much to add at the moment.  The referenced PR that I sent is also very minor.  You can merge it or close it - won't make much difference.  Only when the code grows larger will this refactoring be more important, IMHO.  
For the updates on GDP SDE and convergence... I have some unmerged code lying around but it's all a bit stalled on my side at the moment but maybe we can get together and plan so that we get momentum again.

**Comment by sashko** [comment]
@gunnarx yes, I've seen your PR and deleted my comment. We'll take a look into it ASAP. Sorry for such a stupidly long delay.
